### DISCUSSION TITLE
Parametric centerpieces

### DISCUSSION THREAD

**Discussion by OneMHz** [aderusha/DDD-Printable-Wall-Control-System#23] [Ideas]
I just got a set of leather punches that I wanted to display on my wall control pegboard. It would be great if there was a way to do parametric center-pieces so that I could make a thing to hold them. They have straight sides, so it would need to be a block with holes that don't go all the way through. I'd be interested in doing some modeling/coding for it.

**Comment by fcruz-ehr** [comment]
Yea, I really love this repo, and I have remixed centerpieces into all sorts of other models to put them onto my pegboard - but it's always some hacky stitching together of  STLs in tinkercad.  Do you know if the source sketchup files are parametric? I have  never used sketchup, I just use Fusion 360, but I would be willing to learn sketchup to contribute some mods to this.

**Reply by sl0wfi** [reply]
It seems like a worthwhile effort to create a OpenSCAD centerpiece. I am not terribly skilled, but I think I am going to give it a try. I don't think I can live with SketchUp, or pay for it!

**Reply by benzvan** [reply]
I'm a little late to the party but I started an OpenSCAD library for DDD-Printable-Wall-Control-System here: https://github.com/benzvan/DDD-Printable-Wall-Control-OpenSCAD/tree/main

**Comment by f1yn** [comment]
I just added a PR that adds parametric (FreeCAD) files for Quickhooks and Centerpieces https://github.com/aderusha/DDD-Printable-Wall-Control-System/pull/39
